src/components/Content.js

```diff
@@ -50,11 +55,13 @@ Content.defaultProps = {
     showSearch: true
 };
 
-const mapDispatchToProps = { fetchSearchedMovies };
+const mapDispatchToProps = dispatch => bindActionCreators({
+    fetchSearchedMovies
+}, dispatch);
```

Comment:
`const mapDispatchToProps = { fetchSearchedMovies };` should still work :)
What is the reason of using `bindActionCreators`?